S/SuiteSparse/SuiteSparse32/build_tarballs.jl

```diff
@@ -0,0 +1,68 @@
+using BinaryBuilder
+
+name = "SuiteSparse32"
+version = v"5.4.0"
+
+# Collection of sources required to build SuiteSparse
+sources = [
+    ArchiveSource("https://github.com/DrTimothyAldenDavis/SuiteSparse/archive/v$(version).tar.gz",
+                  "d9d62d539410d66550d0b795503a556830831f50087723cb191a030525eda770"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/SuiteSparse-*
+
+# Disable OpenMP as it will probably interfere with blas threads and Julia threads
+FLAGS+=(INSTALL="${prefix}" INSTALL_LIB="${libdir}" INSTALL_INCLUDE="${prefix}/include" CFOPENMP=)
+
+FLAGS+=(LDFLAGS="${LDFLAGS} -L${libdir}")
+if [[ ${target} == *mingw32* ]]; then
+    FLAGS+=(UNAME=Windows)
+else
+    FLAGS+=(UNAME="$(uname)")
+fi
+
+FLAGS+=(BLAS="-lopenblas" LAPACK="-lopenblas")
+
+# Disable METIS in CHOLMOD by passing -DNPARTITION and avoiding linking metis
+#FLAGS+=(MY_METIS_LIB="-lmetis" MY_METIS_INC="${prefix}/include")
+FLAGS+=(UMFPACK_CONFIG="$SUN" CHOLMOD_CONFIG+="$SUN -DNPARTITION" SPQR_CONFIG="$SUN")
+
```

Comment:
None of these things are being built here. So could just delete the FLAGS block (lines 28-31). Metis won't be looked for either.